lib/nailinda_web/templates/patient/index.html.eex

```diff
@@ -1,5 +1,6 @@
-<b><h2>Patient details</h2></b>
- <table>
+<h2 class="title has-text-centered has-text-weight-bold">Patient details</h2>
+<div class="columns is-centered">
```

Comment:
seems like you forgot to terminate this div tag

Reply:
The div was terminated. You can click on "view file" to see a preview of the whole file :smile_cat: